### DISCUSSION TITLE
Error SQL - Conflict - Special installation

### DISCUSSION THREAD

**Discussion by LuffyyyffuL** [requarks/wiki#6516] [Error / Bug Report]
Hi,

I use WikiJS in a somewhat special way because I have imposed constraints. The application part of WikiJS is launched in Docker on client computers and the database part runs on an external Postgres server. This way of doing things works, but today we encountered the following problem, conflict :

`UPDATE "pagesVector" SET title = $1, description = $2, tokens = (setweight(to_tsvector('french', $3), 'A') || setweight(to_tsvector('french', $4), 'B') || setweight(to_tsvector('french', $5), 'C')) WHERE path = $6 AND locale = $7 - relation "pagesVector" does not exist`

The "pagesVector" table does not exist, I can't understand why I have this problem. If anyone has an idea, I'm a taker.

Thank you in advance

**Comment by mckernanin** [comment]
I am also seeing this on a fresh install using postgres advanced search

**Comment by NGPixel** [comment]
Try to switch to the basic DB Search, save. then switch back to DB Postgres, save. Then run a index rebuild.
